components/python/salt/Makefile

```diff
@@ -0,0 +1,48 @@
+# This file and its contents are supplied under the terms of the
```

Comment:
Missing first license line '#\n'

Reply:
It' s missing in python.mk template aswell

Reply:
Fixed